tests/test_working_endpoints_97.py

```
@@ -2,10 +2,11 @@
 Working tests for app module endpoints – final push to 97%.
 
 RU: Рабочие тесты для покрытия модуля app, не main.py (докстринг
-скорректирован для一致ности с импортами).
+скорректирован для согласованности с импортами).
```

Ruff (0.14.0)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)